src/js/module/main/main.js

```diff
@@ -3,6 +3,8 @@ import MainViewDialog from 'module/main/dialog';
 import Const from 'const';
 import Button from '../component/button';
 
+const IS_PLANTING_CLASS = 'is-planting';
```

Comment:
:+1: for introduction of global variable, I'm just thinking that this class should have slightly different name, eg.: `plantingjs-engine`.